mne/coreg.py

```diff
@@ -1223,3 +1227,576 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
+
+        self._rot_trans = None
+        self._n_scale_params = (0, 1, 3)
+        self._n_scale_param = self._n_scale_params[0]
+        self._coord_frames = ('mri', 'head')
+        self._coord_frame = self._coord_frames[0]
+        self._icp_iterations = 20
+        self._icp_angle = 0.2
+        self._icp_distance = 0.2
+        self._icp_scale = 0.2
+        self._icp_fid_matches = ('nearest', 'matched')
+        self._icp_fid_match = self._icp_fid_matches[0]
+        self._grow_hair = 0.
+        self._lpa_weight = 1.
+        self._nasion_weight = 10.
+        self._rpa_weight = 1.
+        self._hsp_weight = 1.
+        self._eeg_weight = 1.
+        self._hpi_weight = 1.
+
+        self._dig = _DigSource(info)
+        self._mri = _MRIHeadWithFiducialsModel(subjects_dir=subjects_dir,
+                                               subject=subject)
+        self._nearest_calc = self._nearest_calc_default()
+
+    def _update_params(self, params):
+        rot_changed = False
+        if self._rot_trans is None or params[:3] != self._parameters[:3]:
+            rot_changed = True
+            self._last_parameters[:3] = self._parameters[:3]
+            self._parameters[:3] = params[:3]
+            self._rot_trans = rotation(*params[:3]).T
+        tra_changed = False
+        if rot_changed or params[3:6] != self._parameters[3:6]:
+            tra_changed = True
+            self._last_parameters[3:6] = self._parameters[3:6]
+            self._parameters[3:6] = params[3:6]
+            self._head_mri_t = self._rot_trans.copy()
+            self._head_mri_t[:3, 3] = \
+                -np.dot(self._head_mri_t[:3, :3], params[3:6])
+            self._mri_head_t = self._rot_trans.copy()
+            self._mri_head_t[:3, 3] = np.array(params[3:6])
+        if tra_changed or params[6:9] != self._parameters[6:9]:
+            self._last_parameters[6:9] = self._parameters[6:9]
+            self._parameters[6:9] = params[6:9]
+            if self._coord_frame == 'head':
+                self._mri_trans = self._mri_head_t.copy()
+            else:
+                self._mri_trans = np.eye(4)
+            self._mri_trans[:, :3] *= params[6:9]
+
+    @property
+    def _changes(self):
+        new = np.array(self._parameters, float)
+        old = np.array(self._last_parameters, float)
+        move = np.linalg.norm(old[3:6] - new[3:6]) * 1e3
+        angle = np.rad2deg(_angle_between_quats(
+            rot_to_quat(rotation(*new[:3])[:3, :3]),
+            rot_to_quat(rotation(*old[:3])[:3, :3])))
+        percs = 100 * (new[6:] - old[6:]) / old[6:]
+        return move, angle, percs
+
+    @property
+    def _transformed_dig_hpi(self):
+        return apply_trans(self._hsp_trans, self._dig.hpi_points)
+
+    @property
+    def _transformed_dig_eeg_points(self):
+        return apply_trans(self._hsp_trans, self._dig.eeg_points)
+
+    @property
+    def _transformed_dig_points(self):
```

Comment:
```suggestion
    def _transformed_dig_extra(self):
```

would be more consistent.